sas9.4-viya/azuredeploy.json

```diff
@@ -82,6 +82,13 @@
                 "description": "The SAS data volume size for SAS 94"
             }
         },
+        "primaryUserName": {
+            "type": "string",
+            "metadata": {
+                "description": "Username for the Virtual Machine."
+            },
+            "defaultValue": "vmuser"

```

Comment:
remove this defaultValue - increases surface area

Reply:
done. Removed the defaultValue